releasenotes/notes/bump-golang-1.19.4-8a61d93d111cc229.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Agents are now built with Go ``1.19.4``.
```

It feels unusual to mark the version number as an identifier, but feel free to ignore.

```suggestion
    Agents are now built with Go 1.19.4.
```

---

I just followed the pattern used in other changelogs in the past; so I'll keep it for consistency if that's OK.